api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -456,6 +456,18 @@ else
     exit 1
 fi
 
+##########################
+# Update rolled up counts #
+##########################
+echo "Updating rolled up counts in achilles results from criteria (for snomed conditions)"
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"UPDATE \`$OUTPUT_PROJECT.$OUTPUT_DATASET.achilles_results\` ar
+SET ar.count_value = cr1.est_count
```

Comment:
Is this overwriting a previous value in the table? What value was already there?

Reply:
Previously there was the count which is the count of concept by itself and not the rolled up count.

Reply:
Please add this comment to the code